course5/homework02/homework0201.cc

```diff
@@ -13,15 +13,15 @@ The algorithm accepts a 32-bit floating-point number as the input and stores a h
 
 float carmack_sqrt(float input)
 {
-    long* fl_to_int = reinterpret_cast<long*>(&input);
+    int32_t* fl_to_int = reinterpret_cast<int32_t*>(&input);
```

Comment:
These are not really a c++ types, but rater alaises provided by the library.
The proper way of using these is to a) ```#include <cstdint>``` and b) use qualified version: ```std::int32_t```